quickstart.md

```diff
@@ -0,0 +1,55 @@
+# Quickstart Guide
+
+This is a very quick run-down of the [README](README.md) file.
+
+## Before you start
+
+* If in doubt, don't delete any files in this repository
+* If you wish to contribute, please fork this repo and submit changes via Pull Request
+* Please contact us by raising an Issue if you have any question(s)
+
+## Setting up Git and contributing
+
+1. Install Git for your OS
+2. Fork this repo to your GitHub account via the Fork button on the top right corner
+3. Clone the forked repo to your computer:
+```
+git clone git@github.com:<github_username>/i18n.git
+```
+4. Add remote so you can pull changes from original repo:
+```
+git remote add ja git@github.com:swcarpentry-ja/i18n.git
+```
+5. Make changes to your local repo - make sure to commit often:
+```
+git add *
```

Comment:
```suggestion
git add -u
```

I would advise this to update files and avoid adding extra files. Of course we should put anything we don't want added into the `.gitignore` as well.